frontend/www/js/omegaup/components/arena/Course.vue

```diff
@@ -58,5 +68,10 @@ export default class ArenaContestPractice extends Vue {
   get activeProblemAlias(): null | string {
     return this.activeProblem?.problem.alias ?? null;
   }
+
+  onNavigateToProblem(request: ActiveProblem) {
+    this.activeProblem = request;
+    this.$emit('navigate-to-problem', request);
```

Comment:
nit:

```suggestion
  onNavigateToProblem(activeProblem: ActiveProblem) {
    this.activeProblem = activeProblem;
    this.$emit('navigate-to-problem', activeProblem);
```

esto no es necesariamente una petición: es un problema.